.pre-commit-config.yaml

```diff
@@ -34,6 +34,7 @@ repos:
     rev: v0.812
     hooks:
       - id: mypy
+        # Copied from setup.cfg
         exclude: "properties|asv_bench"
```

Comment:
Now that `setup.cfg` has 

```
[mypy]
exclude = properties|asv_bench|doc
```

do we still need this `exclude` option here?

Reply:
We do, unfortunately.

I guess that pre-commit gives it a list of files, and when passed a list of files mypy doesn't honor the exclusions.